src/datagen/main/java/mekanism/common/tag/MekanismTagProvider.java

```diff
@@ -416,4 +417,4 @@ private void addInfuseTags() {
         addToTag(MekanismTags.InfuseTypes.FUNGI, MekanismInfuseTypes.FUNGI);
         addToTag(MekanismTags.InfuseTypes.BIO, MekanismInfuseTypes.BIO);
     }
-}
\ No newline at end of file
+}
```

Comment:
Is there a reason you are changing whether or not there is a new line at end of the different java files?

Reply:
I think my IDE must of have done somehow